Lib/cgi.py

```diff
@@ -115,7 +115,8 @@ def closelog():
 # 0 ==> unlimited input
 maxlen = 0
 
-def parse(fp=None, environ=os.environ, keep_blank_values=0, strict_parsing=0):
+def parse(fp=None, environ=os.environ, keep_blank_values=0,
```

Comment:
Similar to urllib.parse.rst we should document this change in https://github.com/python/cpython/blob/master/Doc/library/cgi.rst